doc/cephadm/certmgr.rst

```diff
@@ -16,27 +16,24 @@ while offering flexibility for deployment preferences.
 Certificate Management Behavior
 ===============================
 
-Cephadm `certmgr` automatically detects whether a certificate is self-signed
-(generated by Cephadm) or user-provided. This distinction determines how it
-handles expirations and renewals:
+Cephadm ``certmgr`` automatically detects whether a certificate is self-signed (generated
+by Cephadm) or user-provided as an embedded value in the spec, or referenced externally.
+This distinction determines how it handles expirations and renewals:
 
 Self-Signed Certificates
 ------------------------
-- `certmgr` can fully automate renewal, ensuring seamless service operation.
+- ``certmgr`` can fully automate renewal, ensuring seamless service operation.
 - Automation is controlled by configuration parameters defining certificate
   duration, renewal thresholds, and whether automated rotation is enabled.
 
 User-Provided Certificates
 --------------------------
-- `certmgr` does not renew these automatically but continuously monitors
-  their status.
-- If an issue is detected (e.g., expiration is approaching), it issues
-  health warnings/errors `CEPHADM_CERT_ERROR` to alert administrators,
-  who must manually replace the certificate.
 
-This proactive approach ensures secure and uninterrupted service operations
-while allowing users to manage certificate policies according to their needs.
+There are two types:
+- Inline: Certificates embedded in the service spec; cannot be modified directly via certmgr.
+- Reference: Certificates set and managed via certmgr; must be manually uploaded.
 
+See ``Certificate Health Monitoring`` for details on how certmgr handles user-provided certificate expiration.
```

Comment:
Suggest a link here instead of inline code.

```suggestion
See `Certificate Health Monitoring`_ for details on how certmgr handles user-provided certificate expiration.
```

Also another occurrence of certmgr without inline code formatting.